src/client/components/MainHeader/MainHeader.tsx

```diff
@@ -32,56 +38,245 @@ function OtapLabel() {
   ) : null;
 }
 
+const AmsMegaMenuClassname = 'ams-mega-menu';
+const BurgerMenuToggleBtnId = 'BurgerMenuToggleBtn';
+
+function isTargetWithinMenu(target: any) {
+  const LinkContainer =
+    document.getElementsByClassName(AmsMegaMenuClassname)[0];
+  const BurgerMenuToggleButton = document.getElementById(BurgerMenuToggleBtnId);
+  return (
+    (LinkContainer && LinkContainer.contains(target)) ||
+    (BurgerMenuToggleButton && BurgerMenuToggleButton.contains(target))
+  );
+}
+
+type Props = {
+  isAuthenticated: boolean;
+};
+
+export function SecondaryLinks() {
+  const { BRP, KVK, PROFILE } = useAppStateGetter();
+
+  const profileType = useProfileTypeValue();
+
+  return (
+    <>
+      <PageMenu.Link href={LOGOUT_URL}>Uitloggen</PageMenu.Link>
+      {!isError(BRP) && !isError(KVK) && (
```

Comment:
Als een van beide een error retourneert wordt dit niet getoond, is dat de bedoeling?

Reply:
zo staat het in main, maar ik wil het wel aanpassen naar een ||